api/dscinitialization/v1/dscinitialization_conversion.go

```diff
@@ -38,13 +38,6 @@ func (c *DSCInitialization) ConvertTo(dstRaw conversion.Hub) error {
 			CustomCABundle:  c.Spec.TrustedCABundle.CustomCABundle,
 		}
 	}
-	if c.Spec.DevFlags != nil {
```

Comment:
i might missed something here.
i thought we only wanna remove devFlags on the components.
service seems did not have real implemenation on it.
but keep dsci with devFlags.
because components might raise secruity concrn, the one on dsci is only for  debug level

Reply:
Ok, from the task I understood we wanted to remove devFlags support completely. I revert the log level configuration, removing all deprecated fields.

Reply:
Done

Reply:
thanks!  

@grdryn should we update description in jira to reflect this.